R/github.R

```diff
@@ -15,18 +15,21 @@
 #' @param pull_requests_json_folder_path The path to the downloaded pull requests JSON. See \code{\link{github_api_project_pull_request}}.
 #' @param comments_json_folder_path The path to the downloaded comments JSON. See \code{\link{github_api_project_issue_or_pr_comments}}.
 #' @param commit_json_folder_path The path to the downloaded commits JSON (used to map github username to the git log). See \code{\link{github_api_project_commits}}.
+#' @param pr_comments_json_folder_path The path to the download pr comments JSON. see \code{\link{github_api_project_pr_comments}}.
 #' @return A single reply table which combines the communication from the three jsons.
 #' @export
 parse_github_replies <- function(issues_json_folder_path,
                                  pull_requests_json_folder_path,
                                  comments_json_folder_path,
-                                 commit_json_folder_path){
+                                 commit_json_folder_path,
+                                 pr_comments_json_folder_path){
```

Comment:
This is a note for future self: This function should likely go away. It is coupled to too many paths. A much saner parser is to just parse each of the files with the same column names.